tests/generator/test_compression.py

```diff
@@ -11,7 +11,7 @@
     simple_solution_generator,
     spend_bundle_to_serialized_coin_spend_entry_list,
 )
-from chia.full_node.generator import run_generator, create_generator_args
```

Comment:
I'm pretty sure these calls were also meant to be to the "unsafe" version rather than the mempool version

Reply:
I agree.